osgtest/tests/test_410_jobs.py

```diff
@@ -12,6 +12,16 @@
 
 class TestRunJobs(osgunittest.OSGTestCase):
 
+    def tearDown(self):
+        env_vars = ('_condor_SCITOKENS_FILE',
+                    'BEARER_TOKEN_FILE')
+
+        for env_var in env_vars:
+            try:
+                os.environ.pop(env_var)
```

Comment:
```suggestion
                del os.environ[env_var]
```
pop() is only if you also want the value back.
Alternatively you could use
```
os.environ.pop(env_var, None)
```
in which case you can drop the KeyError check

Reply:
Fixed! The fewer tabs the better